### DISCUSSION TITLE
[Bug]  CJK characters showing up as red invalid characters

### DISCUSSION THREAD

**Discussion by softmarshmallow** [microsoft/monaco-editor#3674] [Q&A]
### Reproducible in vscode.dev or in VS Code Desktop?

- [X] Not reproducible in [vscode.dev](https://vscode.dev) or VS Code Desktop

### Reproducible in the monaco editor playground?

- [ ] Not reproducible in [the monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html)

### Monaco Editor Playground Code

```typescript
// The Monaco Editor can be easily created, given an
// empty container and an options literal.
// Two members of the literal are "value" and "language".
// The editor takes the full size of its container.

monaco.editor.create(document.getElementById('container'), {
	value: "function hello() {\n\talert('Hello world!');\n}",
	language: 'javascript'
});

이것은 한글 입니다.  // < ---- this line
```

### Actual Behavior

The Korean character should be accepted. regard less to syntax.

Even if it is a valid syntax, it will still show up as red character. e.g. in jsx string `<div>한글</div>`

### Additional Context

![image](https://user-images.githubusercontent.com/16307013/146244822-bb600d06-51a5-476e-8e84-de5ac6f00926.png)

**Comment by rcjsuen** [comment]
@softmarshmallow It will not accept random Korean characters because the syntax is invalid. If you put it in a string it will not flag it as an error.

![image](https://user-images.githubusercontent.com/15629116/146247490-41784c38-44e8-40b6-8884-89988d66a8d2.png)

**Comment by softmarshmallow** [comment]
@rcjsuen It also does not work in jsx/tsx Try tsx / jsx file with this content.

not working - `<div>한글</div>`
works (of-course) `<div>{"한글"}</div>`

**Comment by rcjsuen** [comment]
I don't think the Monaco Editor supports JSX out-of-the-box. See #264.

**Comment by softmarshmallow** [comment]
@rcjsuen What about the part that CJK Characters displaying as a red char? How do I solve that part (regarding less to JSX)
